src/components/reminder-form/reminder-form.tsx

```diff
@@ -3,33 +3,56 @@ import React, { FormEvent, useState } from "react";
 
 const Form = ({ }: {}) => {
   const [taskName, setTaskName] = useState('');
-  const [recurring, setRecurring] = useState(false);
+  const [taskMessage, setTaskMessage] = useState('');
+  const [reminderType, setReminderType] = useState('');
+  const [reminderDate, setReminderDate] = useState('');
+  const [recurring, setRecurring] = useState(true);
+  const [reccuringFrequency, setReccuringFrequency] = useState('');
+  const [reminderBeforeMs, setMs] = useState('');
+  let task = []
   const handleSubmit = (e: FormEvent<HTMLFormElement>): void => {
     e.preventDefault();
+    let taskObject = {
```

Comment:
You can set the task object to be of type task and see the TS magic happen when you mismatch any field